debian/marquee/04-nginx-sites

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+rm -f /etc/nginx/sites-enabled/*
+cp nginx/sites/*.conf /etc/nginx/sites-available/
+
+for domain in `cat DOMAINS | tr , ' '`; do
+    ln -s /etc/nginx/sites-available/$domain.conf /etc/nginx/sites-enabled/
+done
+
+/usr/sbin/nginx -t
```

Comment:
Ditto questions.

Reply:
/usr/sbin/nginx -t

> Ditto questions.

Changed to just `nginx` and `restart` there too